### PR TITLE
feat(demo): add example with named outlets

### PR DESCRIPTION
**Pull request by Reactiver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [x] Documentation content changes

## What is the current behavior?

No documentation how to use routable dialogs with named outlets.

Closes # 

## What is the new behavior?

Added example how to use routable dialog with named outlets.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/demo/src/modules/customization/routable/eager/routable-dialog.component.ts [hunk 1]
```diff
@@ -16,7 +20,35 @@ export class RoutableDialogComponent {
...
+
     readonly addRouterOutlet = import('./examples/setup/add-router-outlet.md?raw');
     readonly importModule = import('./examples/setup/import-module.md?raw');
     readonly useRouteGenerator = import('./examples/setup/use-route-generator.md?raw');
+
+    /**
+     * Helps to destroy router-outlet for named outlet example
+     */
+    readonly isNamedOutletPage$ = this.router.events.pipe(
+        filter(isNavigationEndEvent),
```
Comment (splincode) [inline projects/demo/src/modules/customization/routable/eager/routable-dialog.component.ts:39]: `filter(event => event instanceof NavigationEnd)` why just not?

File: package.json [hunk 2]
```diff
@@ -90,6 +90,7 @@
         "@nrwl/nx-cloud": "15.1.1",
         "@nrwl/tao": "15.8.5",
         "@nrwl/workspace": "15.8.5",
+        "@testing-library/cypress": "^9.0.0",
```
Comment (splincode) [inline package.json:93]: `"9.0.0"` please use fix version, we use pin strategy for renovate

File: projects/demo/src/modules/customization/routable/eager/routable-dialog.component.ts (hunk 1 above)
Comment (Reactiver) [reply projects/demo/src/modules/customization/routable/eager/routable-dialog.component.ts:39]: Got problem with typings. `filter` method just pass previous type(`Event`) as result after filtration.

image

File: package.json (hunk 2 above)
Comment (Reactiver) [reply package.json]: ok, fixed

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/customization/routable/eager/routable-dialog.template.html [hunk 3]
```diff
@@ -14,6 +14,18 @@
         </tui-doc-example>
     </ng-template>
 
+    <ng-template pageTab="NamedOutlet">
+        <p i18n>Also it works with named outlet</p>
```
Comment (vladimirpotekhin) [inline projects/demo/src/modules/customization/routable/eager/routable-dialog.template.html:18]: nit: `It also works ...`

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**